src/javascript/app/pages/cashier/payment_agent_list.js

```diff
@@ -16,6 +16,11 @@ const PaymentAgentList = (() => {
             });
         });
 
+        if (!Client.get('currency')) {
```

Comment:
do we need to check if client is logged in then only check the currency? we should do that if we allow logged out clients to see this page